docs/organizations-and-projects.mdx

```diff
@@ -0,0 +1,15 @@
+# Organizations and Projects
+
+## Organizations
+Organizations are the top-level umbrella for all the projects in a company. A user can create multiple organizations to separate projects under different companies or teams.
+An organization can have multiple users who have access to all projects under the organization. Users can invite other users to join and manage the organization. Check out the [Invite Users](/docs/invite-users) page for more information on inviting users to an organization.
+
+The name of the organization can typically be the name of the company or the department.
+
+Example: **My Company**.
+
+## Projects
+Projects are the workspaces for profile data within an organization, mainly to collect profiles that belong to a specific application or environment. Profiles within a project can be compared and analysed against each other as required.
```

Comment:
Maybe add something like "Projects are the unit of separation, data from two projects cannot be queried across each other."

Reply:
done, added this.